plugins/Live/Controller.php

```diff
@@ -114,10 +120,17 @@ public function getVisitorProfilePopup()
     {
         $idSite = Common::getRequestVar('idSite', null, 'int');
 
+        Piwik::checkUserHasViewAccess($idSite);
```

Comment:
Not sure why above we use Common::getRequestVar... instead of `$this->idSite`

Reply:
Guess it shouldn't make a difference to use `$this->idSite` instead